newdle/client/src/reducers/creation.js

```diff
@@ -139,6 +157,8 @@ export default combineReducers({
   },
   private: (state = false, action) => {
     switch (action.type) {
+      case NEWDLE_RECEIVED:
+        return action.newdle.private;
```

Comment:
Don't you need something like this below for `notify` as well?

Reply:
Good catch, this came from the latest rebase.